api/src/wfl/api/routes.clj

```diff
@@ -122,10 +122,18 @@
   [status message exception request]
   {:status (or (:status (ex-data exception)) status)
    :body {:message (or (.getMessage exception) message)
-          :exception (.getClass exception)
+          :exception (str (.getClass exception))
           :data (ex-data exception)
           :uri (:uri request)}})
 
+(defn unexpected-ex-handler
+  "Like [[ex-handler]] but also logs information about the exception."
```

Comment:
logs -=> log for consistency.
Maybe call it `logging-ex-handler` then?
Maybe rename `ex-*` to `exception-*` too.